### PR TITLE
fix:#1195:Padding and Margin added to viewPager

### LINKED ISSUE #1195: Problem in moving across fragment and holding in the middle between two tabs
**Summary:*

Moving across tabs in payment fragment is a problem,Adjacent pages are not visible
Summarize your issue here.

**Steps to reproduce:**

How can we reproduce again the issue?
Payments fragment->Switch across fragments
**Expected behavior:**

What did you expect the app to do?
It should show adjacent tabs while swiping
**Observed behavior:**

What did you see instead?  Describe your issue in detail here.
While moving across the fragments sometimes the tabs get stuck here and there causing failing in switching the tabs
**Device and Android version:**

What make and model device (e.g., Samsung Galaxy S3) did you encounter this on?  What Android
version (e.g., Android 4.0 Ice Cream Sandwich or Android 6.0 Marshmallow) are you running?  Is it
 the stock
version from the manufacturer or a custom ROM?
samsung SM-M013F

**Screenshots:**
https://user-images.githubusercontent.com/56928954/106397409-2a41d800-6433-11eb-9e31-de11dfb73f82.mp4

Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.

### PR DESCRIPTION
**Pull request by haran2248**
## Issue Fix
Fixes #1195 

https://user-images.githubusercontent.com/56928954/106398029-92de8400-6436-11eb-9908-703b2024d1ef.mp4

## Description

Padding and margin added Viewgroup and Clip padding disabled

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by EGOR-IND — APPROVED** [review]
Nice one, I was actually thinking about this issue yesterday, and you already made it, good:).

**Comment by haran2248** [comment]
@devansh-299 can you review this?

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java [hunk 1]
```diff
@@ -40,7 +40,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
             Bundle savedInstanceState) {
         View rootView = inflater.inflate(R.layout.fragment_finance, container, false);
         ButterKnife.bind(this, rootView);
-
+        vpTabLayout.setPadding(5, 0,5, 0);
```
Comment (devansh-299) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java]: @haran2248 these values as accepted as `pixels` so if you want to set `5dp` which I am guessing you probably are trying to, you need to first convert 5dp to pixels and then pass them here. 
Also always leave a space after `,`  I am not sure how this went pass through the GitHub actions we set up but anyways before making any PR please run `./gradlew check` or `gradlew check` on your machine to detect these errors on your machine only

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java [hunk 2]
```diff
@@ -40,7 +40,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
...
+        vpTabLayout.setClipToPadding(false);
+        vpTabLayout.setPageMargin(12);
```
Comment (devansh-299) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java]: same here also, convert this to pixels first too.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java (hunk 1 above)
Comment (devansh-299) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java]: adding @s-ayush2903 here

**Review by devansh-299 — CHANGES_REQUESTED** [review]

**Comment by haran2248** [comment]
Yeah Ill work on this

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java (hunk 1 above)
Comment (haran2248) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java]: Oh i might have missed out on that,Ill make the corrections

Comment (s-ayush2903) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/PaymentsFragment.java]: @devansh-299 While writing the actions I believed, that these mini-checks are included in our lint job, looks like it isn't so. No worries, it is an easy fix, I can push the changes needed in a new PR, thoughts?

**Pull request merged**